src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,30 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double bucketCapacity;
+    private double armLength;
```

Comment:
The field 'armLength' should have an access modifier to encapsulate its visibility properly. Consider using 'private' to ensure that this field is not directly accessible from outside of the class.